sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/DescribeTableSuite.scala

```diff
@@ -774,6 +673,79 @@ class DescribeTableSuite extends DescribeTableSuiteBase with CommandSuiteBase {
   }
 }
 
+/**
+ * The class contains tests for the `DESCRIBE TABLE` command to check V1 In-Memory
+ * table catalog.
+ */
+class DescribeTableSuite extends DescribeTableSuiteBase with CommandSuiteBase {
```

Comment:
pull up the test cases to the parent suite, so that we also test them with Hive.